modules/virt-installing-virtio-drivers-installing-windows.adoc

```diff
@@ -17,15 +17,23 @@ installation method might differ between versions of Windows. See the
 documentation for the version of Windows that you are installing.
 ====
 
+.Prerequisites
+
+* A storage device containing the `virtio` drivers must be attached to the VM.
```

Comment:
It just occurred to me that we should tell the user where to download the drivers. (virtio-win in RH ecosystem catalog). We could add link in an Additional Resources section.